README.md

```diff
@@ -289,6 +289,27 @@ Default Content (if template is unavailable):
 <p><a href="{{ .ConfirmationURL }}">Change Email</a></p>
 ```
 
+`WEBHOOK_URL` - `string`
+
+Url of the webhook listener server. This will called when events like  `validate`, `signup` or `login` occurs.
```

Comment:
can we call this a webhook `receiver` or `target`?

Reply:
:+1: